action/action-dsl/src/main/java/com/powsybl/action/dsl/ast/IsOverloadedNode.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.action.dsl.ast;
+
+import com.powsybl.commons.PowsyblException;
+
+import java.util.List;
+import java.util.Objects;
+
+
+public class IsOverloadedNode implements ExpressionNode {
+
+    private final List<String> branchIds;
+    private final float reduction;
+
+    public IsOverloadedNode(List<String> branchIds, float reduction) {
+        this.branchIds = Objects.requireNonNull(branchIds);
+        if (branchIds.isEmpty()) {
+            throw new PowsyblException("List of branch should contain at least one branchId");
```

Comment:
IllegalArgumentException is probably better here.

Fix the message: « The list of branch Ids should not be empty »